pkg/cmd/get/probe.go

```diff
@@ -0,0 +1,211 @@
+/*
+Copyright © 2021 The LitmusChaos Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package get
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	models "github.com/litmuschaos/litmus/chaoscenter/graphql/server/graph/model"
+	apis "github.com/litmuschaos/litmusctl/pkg/apis/probe"
+	"github.com/litmuschaos/litmusctl/pkg/utils"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+)
+
+var probesCmd = &cobra.Command{
+	Use:   "probes",
+	Short: "Display list of probes",
```

Comment:
maybe we can write this: "Display a list or a specific probe"
WDYT @SarthakJain26 ?

Reply:
How about "Details of a specific probe" we can avoid using "list"

Reply:
yep, sounds good

Reply:
but we are now majorly listing the probes and giving few details if passed probe_id. We are using describe for that right?